services/config_service.lua

```diff
@@ -1,6 +1,61 @@
+function join(t1, t2)
+  for _, v in pairs(t2) do
+    table.insert(t1, v)
+  end
+  return t1
+end
 
 if service.service == "sigfox" and service.action == "updated" then
-  -- local parameters = Config.getParameters({service = "sigfox"})
+  local currentTime = os.time(os.date("!*t"))
```

Comment:
Should we also do that on the "created" action?

Reply:
You mean `added` action?

Reply:
yes sorry